### PR TITLE
fix: [2.4] Querycoord will trigger unexpected balance task after restart

### PR DESCRIPTION
**Pull request by weiliu1031**
issue: #38606 
pr: #38630

### CODE REVIEW

File: internal/querycoordv2/meta/target.go [hunk 1]
```diff
@@ -19,7 +19,9 @@ package meta
 import (
 	"time"
 
+	"github.com/pingcap/log"
```
Comment (chyezh) [inline internal/querycoordv2/meta/target.go:22]: wrong log package here.

**Review by tedxu — COMMENTED** [review]
/lgtm

**Pull request merged**